logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -265,9 +281,9 @@ abstract class UserSessionScopeCommon(
 
     private val logoutRepository: LogoutRepository = LogoutDataSource(authenticatedDataSourceSet.authenticatedNetworkContainer.logoutApi)
     val listenToEvents: ListenToEventsUseCase
-        get() = ListenToEventsUseCase(syncManager, eventRepository, conversationEventReceiver)
+        get() = ListenToEventsUseCase(syncManager)
```

Comment:
This can be removed in a next PR.

Other UseCases will trigger sync and "listening to events".